tests/fix_client_requests.py

```
@@ -1,56 +1,56 @@
-    new_match = new_match.replace('},', '},\n                consistency_level: ConsistencyLevel::Linearizable,')
-    
-    return new_match
-
-# Apply the replacement
-new_content = re.sub(pattern, replace_client_request, content, flags=re.MULTILINE | re.DOTALL)
-
-# Write back
-with open('src/raft/src/integration_tests.rs', 'w') as f:
-    f.write(new_content)
-
+#!/usr/bin/env python3
```

Ruff (0.14.4)

Shebang is present but file is not executable

(EXE001)